docs/blog/newsletter-october-2020.rst

```diff
@@ -0,0 +1,125 @@
+What we're learning in #learn-tech-writing
+------------------------------------------
+
+Over in the `#learn-tech-writing <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__ book club, they’re reading through *Don’t Make Me Think* by Steve Krug. One recent conversation was about documentation structure design: both for individual pages of documentation, and for doc sites too.
+
+At the page level, it’s about making pages scannable. You can break them up into clearly defined areas using headings, and it also helps to use short paragraphs and lists. One documentarian talked about using images to make pages look less daunting - while you want to avoid adding gratuitous images, they do really help break up pages visually.
+
+For a doc set, how do you make it easy for readers to find what they need? People search for information in different ways: some stick to Google or the site search, some scan and click around. It helps to talk to your users to find out, but you can’t expect them all to work the same way. And we have to be aware that our personal preferences don’t necessarily match those of our readers.
+
+A big point of discussion was on how to divvy up pages. At one extreme is individual short pages for each topic (About the Feature, Using the Feature, Reference for the Feature...). It can mean more clicking around, but DMMT suggests: "It doesn’t matter how many times I have to click as long as every click is a mindless, unambiguous choice."
+
+The other extreme is to have everything on one page, and a table of contents to let readers skim and skip through. For example, like the `Asciidoctor manual <https://asciidoctor.org/docs/user-manual/#glossary>`__. 
+
+There’s no best answer here. It’s a tradeoff between scrolling/ctrl+F vs clicking/searching to find what you need. And it depends on who you’re optimising for. For new users, a huge page can be overwhelming and intimidating; you don’t know what’s important and what you can skip. But once you’ve gotten used to the content, if you’re going to refer back to it a lot, one page can be pretty convenient.
+
+There are tons more interesting ideas flying about in the book club, so do `take a look <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__!
+
+----------------
+From our sponsor
+----------------
+
+This month’s newsletter is sponsored by Microsoft:
+
+.. raw:: html
+
+    <hr>
+    <table width="100%" border="0" cellspacing="0" cellpadding="0" style="width:100%; max-width: 600px;">
+      <tbody>
+        <tr>
+          <td width="75%">
+              <p>
+
+              LWant the simplicity of a source code editor with powerful developer tooling?  Try the lightning fast, lightweight, massively customizeable editor that is taking the world by storm.  Did we mention it’s free and supports MacOS, Linux, and Windows?  To learn more and grab your copy, head here: <a href="https://code.visualstudio.com/docs/editor/whyvscode">https://code.visualstudio.com/docs/editor/whyvscode</a>.
```

Comment:
```suggestion
              Want the simplicity of a source code editor with powerful developer tooling?  Try the lightning-fast, lightweight, massively customizable editor that is taking the world by storm.  Did we mention it’s free and supports MacOS, Linux, and Windows?  To learn more and grab your copy, head here: <a href="https://code.visualstudio.com/docs/editor/whyvscode">https://code.visualstudio.com/docs/editor/whyvscode</a>.
```

Reply:
I'm not sure if we can edit this text, but I did notice a few typos.

Reply:
Totally fine for us to fix stuff here.